### DISCUSSION TITLE
Android 11: oversized icons on map

### DISCUSSION THREAD

**Discussion by kosi2711** [osmandapp/OsmAnd#11688] [Q&A]
Hello,

my device (Motorola G8 Power) received an update to Android 11. Since then all icons (via "configure screen") on the map are much larger as they were with Android 10. Here's a screenshot:

![Screenshot_Motorola-Android11](https://user-images.githubusercontent.com/82325892/118398965-242adc80-b65b-11eb-8a91-97b290d88610.png)

For comparison here's a picture from an old device running Android 9:

![Screenshot_LG-Android9](https://user-images.githubusercontent.com/82325892/118399024-6eac5900-b65b-11eb-9635-f2b15c695f5c.png)

I'm running Osmand~ (F-Droid) version 3.9.5 (but with latest 3.9.10 the behaviour has been the same - downgraded for other reasons).

Haven't found a way to zoom out the icons ... Does anybody know this effect respectively what can be done?

Greets

**Comment by vshcherb** [comment]
That's quite interesting... It's purely Android determined behavior on size of icons. Probably you can change it somewhere in Android settings, so "dp" could be redefined. 
Another idea that now your device is detected as large device
https://github.com/osmandapp/OsmAnd/blob/master/OsmAnd/res/values-large/sizes.xml#L17
https://github.com/osmandapp/OsmAnd/blob/master/OsmAnd/res/values/sizes.xml#L112
